FlowCrypt/Controllers/SetupImap/SetupImapViewController.swift

```diff
@@ -418,12 +418,22 @@ extension SetupImapViewController {
         user.imap?.port = settings.port
         user.imap?.connectionType = settings.connectionType.rawValue
         user.imap?.hostname = settings.hostName ?? ""
+
+        // Used for ui tests
+        if user.email.contains("default@flowcrypt.test") {
+            user.imap?.connectionType = "none"
+        }
```

Comment:
What does "none" here mean and what are the other possible values?

Reply:
We use `providers.json` to pre fill imap/smtp settings. Unfortunately it's not possible be mailcore lib to prefill with `none` value. Only `starttls/ssl: true/false` can be specified. 

![Simulator Screen Shot - iPhone 8 - 2021-08-11 at 20 06 23](https://user-images.githubusercontent.com/13693255/129072445-cf64c5a4-8c62-4d98-b012-32d0b7c37ea5.png)

Reply:
Why not set this through the UI in the test code? It's a UI test after all, it's expected to interact with UI elements during the tests.

Reply:
Without this lines we will prefill with tls selected by default. (returned by default from mailcore).
If we change it after to `none` this will prefill with 143 port (default value from mailcore for none security type).
so then we need to change port as well both for imap and smtp. 

This can be done on UI tests level, but this is a bit more time consuming and less stable

Reply:
![Simulator Screen Shot - iPhone 8 - 2021-08-11 at 21 11 39](https://user-images.githubusercontent.com/13693255/129081773-da505d09-9aa2-4d07-84dd-188bc1ac99c2.png)

Reply:
I understand what you mean. Definitely we should use the UI for this. If it's less stable, we should over time find a way to improve the tests to make them more stable. It's possible for UI tests to be reasonably stable too, it just takes some work.

Reply:
What I worried about that there are a lot of bugs inside xcode with ui tests. 
For example, sometimes password text field should be tapped once to show keyboard and sometimes it works only from second tap. 
```
let passwordTextField = app.tables.secureTextFields["Password"]
passwordTextField.tap()
passwordTextField.tap()
```

Will work on it(will mark as draft for now)

Reply:
Understood. Hopefully we can work around all of this.